Make sure we report success for SNDCTL_DSP_SETDUPLEX.

From: Pierre Ossman <ossman@fefdeb5f-60dc-0310-8127-8f9354f1896f>

(Patch by ZlatkO)

git-svn-id: svn://svn.0pointer.net/pulseaudio/trunk@1422 fefdeb5f-60dc-0310-8127-8f9354f1896f
---

 src/utils/padsp.c |    7 ++++++-
 1 files changed, 6 insertions(+), 1 deletions(-)

diff --git a/src/utils/padsp.c b/src/utils/padsp.c
index dbbb9d5..b894009 100644
--- a/src/utils/padsp.c
+++ b/src/utils/padsp.c
@@ -2109,7 +2109,12 @@ static int dsp_ioctl(fd_info *i, unsigned long request, void*argp, int *_errno)
         case SNDCTL_DSP_GETIPTR:
             debug(DEBUG_LEVEL_NORMAL, __FILE__": invalid ioctl SNDCTL_DSP_GETIPTR\n");
             goto inval;
-            
+
+        case SNDCTL_DSP_SETDUPLEX:
+            debug(DEBUG_LEVEL_NORMAL, __FILE__": SNDCTL_DSP_SETDUPLEX\n");
+	    /* this is a no-op */
+	    break;
+	
         default:
             debug(DEBUG_LEVEL_NORMAL, __FILE__": unknown ioctl 0x%08lx\n", request);
 
